src/components/CardItem/CardItems.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from "react";
+import './CardItems.css';
+
+class CardItem extends Component {
+  render() {
+    const {src,alt,title,brief,key} = this.props ;
+    return (
```

Comment:
Line space missing before return
```suggestion

    return (
```